examples/stateful-goroutines/stateful-goroutines.go

```diff
@@ -28,6 +28,7 @@ type readOp struct {
 	key  int
 	resp chan int
 }
+
```

Comment:
I had these blank line omitted so that the "These readOp and writeOp structs encapsulate..." comment clearly references both of the struct definitions, instead of just `readOp`. Do you feel strongly about adding a line here?

Reply:
Nope, no strong feelings. Wasn't aware what it was for, and the whitespace seemed a bit non-idiomatic. The formatting looks decent still:
 
![KD24fCcytAb](https://user-images.githubusercontent.com/1130906/58919745-17b64f00-86e4-11e9-98d7-210a40836081.png)

But if prefer to keep it as before, no worries - I can roll that part back. Let me know

Reply:
OK yeah let's roll back to no space there. It is less idiomatic code formatting but I do prefer the resulting render.

Reply:
Done